server_tests/test/clientTests.js

```diff
@@ -1285,6 +1299,11 @@ describe("RUST API TEST SUITE", function() {
 
     // BUBL TIMELAPSE
     describe('Testing /osc/commands/execute camera._bublTimelapse command', function() {
+        if (!isBublcam) {
+          this.skip();
```

Comment:
wifi password is an `it()` block, not a `describe()` block. Did you run the tests with `MODEL=osc`, and they all pass / the correct ones are skipped?